liblastlog2/man/ll2_read_all.3.adoc

```diff
@@ -0,0 +1,68 @@
+//po4a: entry man manual
+= ll2_read_all(3)
+:doctype: manpage
+:man manual: Programmer's Manual
+:man source: util-linux {release-version}
+:page-layout: base
+:lib: liblastlog2
+:firstversion: 2.39
+
+== NAME
+
+ll2_read_all -  Reads all entries from database and calls the callback function for each entry.
+
+== SYNOPSIS
+
+*#include <lastlog2.h>*
+*int ll2_read_all (const char *__lastlog2_path__,
+     		   int (*__callback__)(const char *__user__, int64_t __ll_time__,
+				       const char *__tty__, const char *__rhost__,
+				       const char *__pam_service__),
+ 		   char **__error__);*
+
+== DESCRIPTION
+
+Reads all entries from database _lastlog2_path_ and calls callback fuction _callback_ for each entry.
+
+--------------------------------------
+char  *error = NULL;
+const char *db_path = "/var/lib/lastlog/lastlog2.db";
+const char *user = "root";
+
+static int
+callback (const char *res_user, time_t ll_time, const char *res_tty,
```

Comment:
Above the `ll_time` parameter is `int64_t`, here it is `time_t`.

Reply:
Yes, I have had taken the example from lastlog2.c which has also been wrong and I have fixed afterword:
https://github.com/util-linux/util-linux/pull/2508/commits/87f8b2eb3f12c7cbbd68c03d9f2d131d2eac6ddf